pkg/api/login.go

```diff
@@ -39,20 +39,45 @@ func LoginView(c *m.ReqContext) {
 		viewData.Settings["loginError"] = loginError
 	}
 
+	if tryOAuthAutoLogin(c) {
+		postLoginRedirect(c)
```

Comment:
This code will write twice to the ResponseWriter. first in `tryOauthAutoLogin` then in `postLoginRedirect` we redirect should only be written once.

Reply:
Thanks for the hint. The problem should be fixed now.